etc/kayobe/ansible/monasca_ceph_monitoring.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Deploy Monasca-rsyslog on Ceph nodes
+  hosts: ceph
+  become: true
+  roles:
+  - role: stackhpc.monasca-rsyslog
+    monasca_rsyslog_api_auth:
+      auth_url: "http://{{ ilab_vip_address }}:5000"
+      project: "monasca"
+      username: "monasca-agent"
+      password: "{{ secrets_monasca_agent_password }}"
+      region_name: "RegionOne"
+      endpoint_type: "public"
+    monasca_rsyslog_venv: "/usr/libexec/monasca-rsyslog"
+
+- name: Deploy Monasca Agent
+  hosts: ceph
+  become: true
+  roles:
+  - role: stackhpc.monasca-agent
+    keystone_url: "http://{{ ilab_vip_address }}:5000/v3"
+    monasca_user: root
+    monasca_agent_user: "monasca-agent"
+    monasca_agent_password: "{{ secrets_monasca_agent_password }}"
+    monasca_agent_project: "monasca"
```

Comment:
I think historically we have pushed these to the p3 project - please check in the existing config file on the ceph nodes.